### DISCUSSION TITLE
Bypass buffering and stuttering audio

### DISCUSSION THREAD

**Discussion by troffasky** [AlexandreRouma/SDRPlusPlus#441] [General]
Was going to log an issue about this but whilst searching the GUI for the version number I found the "Bypass buffering" option which fixed the problem. I can't find anything when I google this so I thought I'd open a discussion instead.
I am using an SDRplay RSP1a plugged in to a remote Pi 4 running Soapy. SDR++ is running on a client machine with Ubuntu 21.04.
If I don't tick "Bypass buffering", the audio stutters 2-3 times per second. The waterfall never stutters so I am guessing the buffering thing is about audio buffering, not network buffering?

**Comment by AlexandreRouma** [comment]
That's just a side effect, it's a RtAudio bug and here's the fix:
https://github.com/AlexandreRouma/SDRPlusPlus/discussions/112
It's the pinned in the discussion section.
